Explorer/Assets/DCL/Navmap/PlacesAndEventsPanelController.cs

```diff
@@ -22,18 +23,23 @@ public PlacesAndEventsPanelController(
             NavmapSearchBarController searchBarController,
             SearchResultPanelController searchResultController,
             PlaceInfoPanelController placeInfoPanelController,
-            EventInfoPanelController eventInfoPanelController)
+            EventInfoPanelController eventInfoPanelController,
+            NavmapZoomController navmapZoomController)
         {
             this.view = view;
             this.searchBarController = searchBarController;
             this.searchResultController = searchResultController;
             this.placeInfoPanelController = placeInfoPanelController;
             this.eventInfoPanelController = eventInfoPanelController;
+            this.navmapZoomController = navmapZoomController;
 
             view.CollapseButton.gameObject.SetActive(true);
             view.ExpandButton.gameObject.SetActive(false);
             view.CollapseButton.onClick.AddListener(Collapse);
             view.ExpandButton.onClick.AddListener(Expand);
+
+            view.PointerEnter += DisableMapZoom;
+            view.PointerExit += EnableMapZoom;
```

Comment:
can you unsub in the dispose?

Reply:
Do you find it necessary? This controller is just created once during the whole lifecycle of the app and i dont see that is going to change